### ISSUE TITLE
Accessibility: Screen readers cannot access current line number

### ISSUE THREAD

**Issue by zersiax** [Microsoft/vscode#64943]
- VSCode Version: 1.29.1
- OS Version: Windows 10 1803

Steps to Reproduce:

1. Open any file while line numbers are enabled in the settings.
2. Try to find out current line number with screen reader.

Does this issue occur when all extensions are disabled?: Yes
